### PR TITLE
Missing import in check_location

### PR DESCRIPTION
**Pull request by sww314**
Fix missing import needed in #520

### CODE REVIEW

**Comment by jschneier** [comment]
@jdufresne @sww314 how do we feel about codecov? I added it in a pique of interest at some point, anything you guys think works better or that we are missing?

File: storages/utils.py [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import posixpath
 
 from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
 from django.utils.encoding import force_text
 
 
+
```
Comment (jdufresne) [inline storages/utils.py]: This extra new line is causing flake8 to fail. If you remove it Travis should be green again.

**Comment by jdufresne** [comment]
@jschneier I have no strong opinion. I think most PRs -- especially features and bug fixes -- should come with tests. But I don't find the coverage report all the useful personally. I'm ok either way.

**Comment by sww314** [comment]
@jschneier I agree with Jon. Tests are critical for features and bug fixes. I do not pay much attention to the code coverage report.

**Pull request merged**